diff --git a/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/Bridge.java b/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/Bridge.java
index f6c257d..2018805 100644
--- a/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/Bridge.java
+++ b/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/Bridge.java
@@ -531,9 +531,11 @@ public class Bridge {
         return;
       }
 
+      /**
       Logger.verbose("callback: " + call.getCallbackId() +
           ", pluginId: " + plugin.getId() +
           ", methodName: " + methodName + ", methodData: " + call.getData().toString());
+      **/
 
       Runnable currentThreadTask = new Runnable() {
         @Override
